### PR TITLE
Add a vscode extension

### PR DESCRIPTION
**Pull request by rictic**
It just loads our schema when opening a package.json file

Fixes https://github.com/google/wireit/issues/82

### CODE REVIEW

File: vscode-extension/.vscodeignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.vscode
+.vscode-test
+.wireit/**
```
Comment (aomarks) [inline vscode-extension/.vscodeignore:3]:
```suggestion
.wireit
```
```suggestion
.wireit/**
```

Comment (aomarks) [inline vscode-extension/.vscodeignore:3]:
```suggestion
.wireit
```

Comment (aomarks) [inline vscode-extension/.vscodeignore:3]:
```suggestion
.wireit
```

File: vscode-extension/package.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+  "name": "wireit-extension",
+  "displayName": "wireit",
+  "description": "For the wireit script runner",
+  "publisher": "todo-register-publisher",
```
Comment (aomarks) [inline vscode-extension/package.json:5]: Where will we publish this? Is there an equivalent to the google org we can use in the vscode extension marketplace? Do we make our own?

Comment (aomarks) [inline vscode-extension/package.json:5]: Where will we publish this? Is there an equivalent to the google org we can use in the vscode extension marketplace? Do we make our own?

File: vscode-extension/package.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        "build:copy-schema",
+        "build:ts"
+      ]
+    },
+    "build:copy-schema": {
+      "files": [
+        "../schema.json"
+      ],
+      "output": [
+        "./schema.json"
```
Comment (aomarks) [inline vscode-extension/package.json:41]:
```suggestion
        "schema.json"
```
```suggestion
        "./schema.json"
```

Comment (aomarks) [inline vscode-extension/package.json:41]:
```suggestion
        "schema.json"
```
```suggestion
        "./schema.json"
```

File: vscode-extension/package.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      "files": [
+        "tsconfig.json",
+        "src/**/*.ts"
+      ],
+      "output": [
+        "out",
+        ".tsbuildinfo"
+      ],
+      "clean": "if-file-deleted",
+      "command": "tsc --build"
```
Comment (aomarks) [inline vscode-extension/package.json:55]:
```suggestion
      "command": "tsc --build --pretty"
```
```suggestion
      "command": "tsc --build"
```

Comment (aomarks) [inline vscode-extension/package.json:55]:
```suggestion
      "command": "tsc --build --pretty"
```
```suggestion
      "command": "tsc --build"
```

File: vscode-extension/package.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      ],
+      "files": [
+        "package.json",
+        "schema.json"
+      ],
+      "output": [],
+      "command": "node ./lib/test/scripts/runner.js"
+    }
+  },
+  "repository": {
```
Comment (aomarks) [inline vscode-extension/package.json:82]: Just a nit but I feel like author, license, bugs, homepage, etc. almost always are at the top.

Comment (aomarks) [inline vscode-extension/package.json:82]: Just a nit but I feel like author, license, bugs, homepage, etc. almost always are at the top.

**Review by aomarks — APPROVED** [review]
:tada:

**Comment by aomarks** [comment]
(For some reason GitHub duplicated all of my comments -- cleaned them up).

File: vscode-extension/package.json (hunk 5 above)
Comment (rictic) [reply vscode-extension/package.json:82]: true but I almost never want them to be, I want to see how do I use this dang thing

maybe I'm weird about that though

File: vscode-extension/package.json (hunk 2 above)
Comment (rictic) [reply vscode-extension/package.json:5]: All good questions! Will investigate tomorrow

**Pull request merged**